myosuite/envs/myo/myochallenge/run_track_v0.py

```diff
@@ -282,10 +287,12 @@ def _maybe_sample_terrain(self):
             self.sim.model.geom_pos[self.sim.model.geom_name2id('terrain')] = np.array([0, 0, -10])
 
     def _randomize_position_orientation(self, qpos, qvel):
-        orientation = self.np_random.uniform(0, 2 * np.pi)
-        euler_angle = quat2euler(qpos[3:7])
-        euler_angle[-1] = orientation
-        qpos[3:7] = euler2quat(euler_angle)
+        orientation = self.np_random.uniform(np.deg2rad(-125), np.deg2rad(-60))
+
+        euler_angle = self.get_intrinsic_EulerXYZ(qpos[3:7]) # Roll, Pitch, Yaw format
```

Comment:
how is this different from `myosuite.utils.quat_math.quat2euler` https://github.com/MyoHub/myosuite/blob/e908da72dffab5192e7530a2a58147c94af7808f/myosuite/utils/quat_math.py#L152
if it is the same function, please use the default one

Reply:
It is different because the math function uses an extrinsic Euler function, so you always have to orient it with respect to the world frame, meaning the axis changes according to the facing direction

The one I had is an intrinsic Euler function that orients the Euler angles according to the pelvis forward facing frame, so the axis is always consistent

Reply:
it might still be good to have them together in the same `utils` describing what is different

Reply:
Should this be in a separate PR, since it affects all the other functions?

Reply:
you can add those function to `utils/quat_math.py` and keep it in this PR

Reply:
Functions added.